src/test/java/org/jenkinsci/plugins/workflow/job/properties/QueryingMockTrigger.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
```

Comment:
@abayer was in travelling and i found his changes good, so opened PR for review. Now i can't even review this PR because i become author.